pkgs/development/libraries/libnfs/default.nix

```
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchFromGitHub, autoreconfHook }:
+{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, enableMultithreading ? false }:
```

```suggestion
{ lib, stdenv, fetchFromGitHub, fetchpatch, autoreconfHook, enableMultithreading ? true }:
```
Do we need the option at all? If not we might as well remove it.

---

@SuperSandro2000 

Thanks for the review.

> Do we need the option at all? If not we might as well remove it.

Enabling multi-threading by default adds a bunch of conditions here and there and is not enabled by library's author by default for performance reasons. But I think the performance difference, if any, should be negligible. So let me redo the PR and enable `pthread` support by default.